src/thread_event.c

```diff
@@ -18,6 +18,16 @@ static void thread_##event##_event_handler(tsd_t *tsd);
 ITERATE_OVER_ALL_EVENTS
 #undef E
 
+static void
+thread_tcache_gc_event_handler(tsd_t *tsd) {
+	assert(opt_tcache);
+	assert(tcache_gc_event_wait_get(tsd) == 0U);
+	if (TCACHE_GC_INCR_BYTES > 0U && tcache_available(tsd)) {
+		thread_tcache_gc_event_update(tsd, TCACHE_GC_INCR_BYTES);
+		tcache_event_hard(tsd, tsd_tcachep_get(tsd));
```

Comment:
let's use `tcache_get` here. The tcachep getter should not be used by other modules; the branch is guarded with tcache_available already.